Update monorepo package.jsons and documentation (#31623)
Automattic/wp-calypso: docs/monorepo.md

	"license": "GPL-2.0-or-later",
	"repository": {
		"type": "git",
		"url": "git+https://github.com/Automattic/wp-calypso.git",

> 		"directory": "packages/your-package"

Reviewer: Is this intended?

Reviewer: I guess it make sense.

Author: Yap, it's an example file 👍

## Publishing

Please do not use regular [`npm publish`](https://docs.npmjs.com/cli/publish) within a package to publish an individual package; `npx` has issues using this flow.

Using [Lerna](https://lernajs.io/) to publish package(s):

> 1. Login to NPM: `npm whoami`, `npm login`.

Author: good point! https://github.com/Automattic/wp-calypso/pull/31623/commits/fbcc91d8f8a4704f734f64db98fcf89dac657900

1. Update packages versions as necessary. We’ll rely on package versions for Lerna to know what to publish. Please be mindful about [semantic versioning](https://semver.org/).
1. `git checkout master`
1. `git pull`
1. `git status` (should be clean!)
1. `npm run distclean`
1. `npm ci`

> 1. `npx lerna publish --dist-tag next from-package`

Reviewer 2: ```suggestion
1. `npx lerna publish from-package`
```

> 1. Lerna will confirm which packages and versions will be published. `--dist-tag next` is optional, we used it to avoid publishing a new latest for `i18n-calypso`. Say “no” at the prompt.

Reviewer 2: ```suggestion
1. Say “no” at the prompt.
1. Lerna will confirm which packages and versions will be published. If something looks off, abort!
1. Make sure you're logged in at this point, we're going to publish 🚀
1. Craft the following command, we'll add `--yes` to skip prompts and save OTP cycle time. `--dist-tag next` is optional, use it when publishing unstable versions.
1. Wait for your npm OTP (one time password) cycle to start, write it into the command and publish:
```

> 1. If everything looked good, get this command ready and wait for your NPM OTP (one time password) cycle to start. `--yes` is added to then end to skip the prompt, we already verified which packages will be published and don’t want to waste the OTP cycle time at the prompt.

Reviewer 2: ```suggestion
```
